scarb/src/bin/scarb/commands/cache_path.rs

```diff
@@ -0,0 +1,9 @@
+use anyhow::Result;
+use scarb::core::Config;
+
+use scarb::ops;
+
+#[tracing::instrument(skip_all, level = "info")]
+pub fn run(config: &Config) -> Result<()> {
+    ops::cache_path(config)
```

Comment:
Similarly to https://github.com/software-mansion/scarb/blob/main/scarb/src/bin/scarb/commands/manifest_path.rs, I do not think we need to move this to `ops`.